packages/mds-policy-linter/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@mds-core/mds-policy-linter",
+  "version": "0.1.0",
+  "description": "Linter tool for MDS Policy documents",
+  "keywords": [
+    "policy",
+    "lint"
+  ],
+  "license": "Apache-2.0",
+  "author": "Marie Maxham",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc --build tsconfig.build.json",
+    "start": "pnpm watch index.ts",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "pnpm build && nodemon --watch '../../packages' --ext 'ts' --ignore '*.d.ts' --exec pnpm ts-node --"
+  },
+  "dependencies": {
+    "@mds-core/mds-policy-service": "workspace:0.11.26",
+    "@mds-core/mds-types": "workspace:0.9.2",
+    "@mds-core/mds-utils": "workspace:0.5.0",
+    "ajv": "8.10.0",
```

Comment:
I think this is an advanced version of `ajv` compared to what we're currently using elsewhere in this repo.